custom_components/ocpp/api.py

```diff
@@ -490,16 +490,18 @@ async def trigger_boot_notification(self):
             return False
 
     async def trigger_status_notification(self):
-        """Trigger a status notification."""
-        req = call.TriggerMessagePayload(
-            requested_message=MessageTrigger.status_notification
-        )
-        resp = await self.call(req)
-        if resp.status == TriggerMessageStatus.accepted:
-            return True
-        else:
-            _LOGGER.warning("Failed with response: %s", resp.status)
-            return False
+        """Trigger status notifications for all connectors."""
+        return_value = True
+        for id in range(0, 1):
```

Comment:
Use `self._metrics[cdet.connectors.value].value` here instead?

Reply:
It does need to be known, so the number of connectors should be retrieved before sending the triggers.